.gitmodules

```
@@ -14,3 +14,6 @@
 [submodule "deps/googletest"]
 	path = deps/googletest
 	url = https://github.com/google/googletest.git
+[submodule "deps/zlib"]
+	path = deps/zlib
+	url = https://github.com/madler/zlib.git
```

I tend to use dev branch. Not sure if it makes sense. But ignore dirty makes sense as the zlib cmake scripts modifies some files
```suggestion
	url = https://github.com/madler/zlib.git
	branch = develop
	ignore = dirty
```

---

after some git fighting i think i changed it :)